### PR TITLE
Improve Test

### PR DESCRIPTION
**Pull request by rngtng**
Thanks to suggestion from @AnWeber test got simpler as well as the implementation.

### CODE REVIEW

File: src/onRequestHook.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  function hasContentTypeKey(headers) {
+    if (headers) {
+      console.log(headers);
+      const entry = Object.entries(headers).find(([key]) => key.toLowerCase() === contentTypeKey.toLowerCase());
+      if (entry && entry.length > 1) {
+        return true;
+      }
+    }
+    console.log("GALSE");
```
Comment (AnWeber) [inline src/onRequestHook.js:13]: Forgotten console.log?

File: src/onRequestHook.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    return false;
+  }
+
+  function hasYamlHeader(string) {
+    return string.split('\n')[0] === '---';
+  }
+
+  if (!hasContentTypeKey(request.headers) && hasYamlHeader(request.body)) {
```
Comment (AnWeber) [inline src/onRequestHook.js:21]: Test if body is string. Could be also buffer or array

**Review by AnWeber — COMMENTED** [review]
Great plugin. Love the idea 👍🏻

File: src/onRequestHook.js (hunk 1 above)
Comment (rngtng) [reply src/onRequestHook.js:13]: yes, classic :D

**Pull request merged**